src/main/java/seedu/clinic/model/product/Product.java

```diff
@@ -9,7 +9,7 @@
 import seedu.clinic.model.attribute.Tag;
 
 /**
- * Represents Product sold by Suppliers or stored in Warehouse.
+ * Represents Product sold by a Suppliers or stored in Warehouse.
```

Comment:
Minor typo here! Should be "Supplier"!

Reply:
Opps it wasn't meant to be changed but I forgot to change it back